css/style.css

```diff
@@ -12,6 +12,7 @@ html {
 body {
   font-weight: normal;
   color: #091e42;
+  font-size: 20px;
 }
 
```

Comment:
- Kindly add @media screen and (min-width: 1080px) into your project. Read again the project requirements.

Reply:
![mediaquery](https://user-images.githubusercontent.com/6615394/126800819-bc7b1689-bf69-437a-9e9c-e75f2d8b9acb.PNG)